packages/libraries/core/src/components/ods-component-attributes-2-string-attributes.ts

```diff
@@ -23,7 +23,7 @@ export function OdsComponentAttributes2StringAttributes<T>(attributes: OdsCompon
     .map(([name, value]) => {
       if (typeof value === "boolean") {
         if (value) {
-          parameters[name] = undefined;
+          parameters[name] = "true";
```

Comment:
with undefined the `circle` was undefined in the stencil button component during the tests